Geometry_Engine/Query/ClosestPoint.cs

```diff
@@ -181,11 +253,14 @@ public static Point ClosestPoint(this PlanarSurface surface, Point point)
             return (panelOutline.IIsContaining(cPt)) ? cPt[0] : panelOutline.IClosestPoint(cPt[0]);
         }
 
-
         /***************************************************/
         /**** Public Methods - Interfaces               ****/
         /***************************************************/
 
+        [Description("Finds the closest point on the geometry in relation to the provided Point.")]
+        [Input("geometry", "The geometry on which to find the closest point in relation to the provided point.")]
+        [Input("point", "The reference point. The point on the geometry with the smallest distance to this point will be returned.")]
+        [Output("closePt", "The point on the geometry with the smallest distance to the provided point.")]
```

Comment:
No fun that the tolerance is not passed here

Reply:
Not disagreeing. Just did not want to interfere to much with other methods in this PR, with versioning etc.

Can ofc add it in this PR, but would almost like to handle it separately. We really do need to run through all methods though and ensure that the tolerance is added as a input parameter when required, as well as properly passed.

If you are fine with it, I would prefer to do it as a separate exercise though.

Reply:
Sure thing, I only made this comment to highlight the fact and add it to the master to do list - absolutely not asking to fix it here 👍